notebooks/2022-18-03inactive_users.ipynb

```diff
@@ -0,0 +1,131 @@
+{
```

Comment:
If I understand this code right, data of potential bots are being written to csv. In that case, better name for the file would be `2022-18-03potential_bots_to_csv.ipynb`